<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideshowProperties</class>
 <widget class="QWidget" name="SlideshowProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slideshow properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="buddy">
        <cstring>titleEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Images:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTreeView" name="slideListView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>84</number>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>16</number>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>84</number>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>16</number>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="KPushButton" name="addButton">
          <property name="toolTip">
           <string>Add (Alt+ Insert)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="shortcut">
           <string>Alt+Ins</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="removeButton">
          <property name="toolTip">
           <string>Remove (Alt + Delete)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="shortcut">
           <string>Alt+Del</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KPushButton" name="upButton">
          <property name="toolTip">
           <string>Move up (Alt + Up)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="shortcut">
           <string>Alt+Up</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="downButton">
          <property name="toolTip">
           <string>Move down (Alt + Down)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="shortcut">
           <string>Alt+Down</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="infoLabel">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Slide duration:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>durationSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="durationSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="specialValueText">
        <string>fit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="addOriginalsCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add to DVD</string>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="loopCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Loop slideshow</string>
       </property>
       <property name="shortcut">
        <string>Alt+L</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Image files:</string>
       </property>
       <property name="buddy">
        <cstring>addOriginalsCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>After last image:</string>
       </property>
       <property name="buddy">
        <cstring>loopCheckBox</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>227</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="KPushButton" name="audioButton">
     <property name="text">
      <string>Audio</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
 <pixmapfunction>BTN_ICON</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global">kpushbutton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
